### DISCUSSION TITLE
Support to highlight `new-line` inside the code block

### DISCUSSION THREAD

**Discussion by gargajit** [fastn-stack/fastn#827] [Q&A] [answered]
For documentation purposes, we want to highlight the requirement of adding a `newline character` in the code block. But we cannot highlight it without adding content to it which is displayed in the code block.

For example:

![image](https://user-images.githubusercontent.com/118595104/229993810-547f3733-d8ab-4ef0-b532-4e534c0bad10.png)

**Comment by amitu** [comment]
@gargajit, Adding a `\;; <hl>` on it's own line is not working? Check Ker ke batao.

**Reply by gargajit** [reply]
@amitu, I tried and it works. 

Earlier I gave a space `\ ;; <hl>` that's why it was giving me `\` as output.

**Comment by gargajit** [comment, marked as answer]
Solution:

Add **`\;; <hl>`** and it highlights the new line. 

Note: Make sure not to give space between `\` and `;;`.
